zerver/lib/bugdown/__init__.py

```diff
@@ -918,6 +918,15 @@ def handleMatch(self, m):
         return ret
 
 class Bugdown(markdown.Extension):
+    def __init__(self, *args, **kwargs):
+        # define default configs
+        self.config = {
+            "realm_filters": ['filters', "Realm-specific filters for domain"],
+             "realm": ['domain', "Realm name"]
+        }
```

Comment:
What is your intent with the default values here?  I don't think the string "filters" and "domain" are valid default values...